Makefile

```diff
@@ -98,14 +98,6 @@ docker-images:
 # Rules for verification, formatting, linting, testing and cleaning #
 #####################################################################
 
-.PHONY: install-requirements
```

Comment:
Can you rebase your branch?
This https://github.com/gardener/gardener-extension-provider-alicloud/pull/516 is already merged.

Reply:
done